deny.toml

```diff
@@ -1,7 +1,7 @@
 targets = [
     { triple = "x86_64-unknown-linux-gnu" },
     { triple = "x86_64-apple-darwin" },
-    { triple = "x86_64-pc-windows-msvc" },
+    { triple = "x86_64-pc-windows-msempty_vec" },
```

Comment:
I think this was an errant find-replace problem? `pc-windows-msempty_vec` sounds like an interesting platform, but not one that Rust supports...

Reply:
yes and its outdated, I noticed it and fixed it